app/helpers/form_helper.rb

```diff
@@ -304,16 +304,24 @@ def help_inline(inline, error)
   def add_label options, f, attr
     label_size = options.delete(:label_size) || "col-md-2"
     required_mark = check_required(options, f, attr)
-    label = options[:label] == :none ? '' : options.delete(:label)
-    label ||= ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
-        s_(clazz.gettext_translation_for_attribute_name attr)) if f
-    label = label.present? ? label_tag(attr, "#{label}#{required_mark}", :class => label_size + " control-label") : ''
+    if options[:label] == :none
```

Comment:
If you want to not touch other functions you can short-circuit at the top and `return '' if options[:label] == :none` as the first line so nothing changes otherwise